docs/comparisons/opa.md

```diff
@@ -0,0 +1,204 @@
+created within the Namespace they belong to.
+Because of that, non-admin Kubernetes users could be allowed
+the RBAC privileges to manage `AdmissionPolicy` resources inside of the
+Namespaces they have access to.
+
+This allows Kubernetes administrators to delegate some policy-related work.
+
+## Policy distribution
+
+The source code of the policy (the Rego code) has to be written inside of
+the Custom Resource that defines a policy inside of Kubernetes.
+
+Kubewarden policies are managed like container images. Once built they are pushed
+into container registries as OCI artifacts.
+
+Kubewarden policies can be signed and verified using container images tools
+like `cosign`, from the [Sigstore project](https://sigstore.dev).
+
+Kubewarden policies can be distributed among geographically distributed container
+registries using the traditional tools and processes adopted for container images.
+
+## CI/CD integration
+
+Both OPA Gatekeeper and Kubewarden can be managed using GitOps methodologies.
+
+However, in the context of OPA Gatekeeper, there's a coupling between the policy's source code
+(the Rego code) and the Custom Resource used to deploy it inside of Kubernetes.
+This introduce some extra steps inside of CI/CD pipelines.
+
+Rego has [testing tools](https://www.openpolicyagent.org/docs/latest/policy-testing/)
+that allow the creation of unit tests suites. Writing tests and executing them inside of
+a CI/CD pipeline is essential to ensure policies behave as expected.
+
+To use these testing tools, the source code of the policy must be made available
+inside of dedicated text files. It's not possible to read the source code from the YAML
+files used to declare the OPA Gatekeeper policy.
+The CI/CD pipeline must keep in sync the Rego source code being tested with the code
+defined inside of the OPA Gatekeeper Custom Resource. This can be done using some 3rd
+party tools.
+
+Kubewarden policies have CI/CD pipelines like the ones of traditional microservices.
+Usually their source code lives inside of a Git repository and then, using
+traditional CI/CD systems, unit tests are ran against it. The unit tests are
+written using the testing frameworks of the language used to write the policy.
+Once all the tests are passing the policy is compiled to WebAssembly and pushed
+to a container registry.
+This kind of pipeline is usually maintained by the policy author.
+
+Kubernetes administrators typically maintain other automation pipelines that react to
+new releases of the policy (leveraging automation tools like
+[Dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot),
+[Renovate bot](https://www.mend.io/renovate/),
+[updatecli](https://www.updatecli.io/) and others) or to changes to the
+policy configuration.
+
+The pipeline tests the policy against different types of requests. The testing can be done using
+the [kwctl](https://github.com/kubewarden/kwctl) cli tool, without requiring a running
+Kubernetes cluster.
```

Comment:
Is it not worth mentioning that the `kwctl` uses the same runtime used in production? I believe this will explicit that `kwctl` is a good tool to test the policies as close to production as possible.

Reply:
+1 here.

Reply:
done